torch/csrc/autograd/variable.h

```diff
@@ -605,15 +555,11 @@ inline Variable make_variable(
     at::Tensor data,
     Edge gradient_edge,
     bool allow_tensor_metadata_change = true) {
-  AT_CHECK(
```

Comment:
also here, I think we want this check for now (although I think the message needs to be updated maybe?).